### ISSUE TITLE
Drupal: Update police page for MVP showing contact information and FAQs

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#16335]
## Description or Additional Context 
In 12/6 sprint planning, it was decided that MVP for the police page would include:
- system page listing police contact information for facilities in that system
  - based on what the data gives us to work with, this will likely be a phone field or use the phone paragraph with unlimited cardinality.
- ~FAQs~ no change needed for this one.

This ticket is to update Drupal to include only that information for MVP.

**Will go in two PRs**
- [x] 1) #16360 adding field to unblock FE  (this PR)
- [x] 2) add migration (separate PR) #16430

## Acceptance Criteria
- [x] Contact list csv is formatted and cleaned up.
  - [x] For contact number use the Telephone Ops column. One number per system for MVP.
  - [ ] API ID column is added.
- [x] Appropriate field is added to the VA Police content type
- [ ] migration is written to populate all the system pages from the CSV (approach to be determined based on the csv structure.

**Comment by swirtSJW** [comment]
Discuss with @davidmpickett as to the field types for the data, keeping in mind two possibly opposing directions:
* simple phone and title field 
   * easiest to implement
   * easiest to sync up on subsequent migrations 
   * easiest for FE to pull into their react data
* phone number paragraph
   * syncing paragraphs in subsequent runs of a migration is challenging
   * pulling that data into the react data is a bit more challenging due to entity references.

**Comment by xiongjaneg** [comment]
@swirtSJW @omahane Example of contact info from Police
[VAP Chief - Facility Contact List (12-5-2023).csv](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13590582/VAP.Chief.-.Facility.Contact.List.12-5-2023.csv)

**Comment by davidmpickett** [comment]
Took a first pass of cleaning up the CSV and indicating the most likely VAMC System and Facility ID string* for each line item. I also found any systems that have multiple line items are marked them as either **good duplicate** (aka there are multiple entries, but they all have the same phone number) or **needs decision**. Looks like there at 10 VAMC systems with multiple entries where we will need our police stakeholders to make a decision on which one to show for mmmVP
 
[New Source of Truth for cleaned CSV](https://dvagov.sharepoint.com/:x:/s/SitewideFacilitiesTeam/ESAtQkMMoh5MhGqhwARq0-QBYhFRLAL0m8vWP_zqlOpgmw?e=RcKb7i)

[Static copy for convenience](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13592153/police_system_contacts.csv)

**Comment by swirtSJW** [comment]
> Will this need UX and a11y review

@xiongjaneg Ideally yes, but given time frame and the fact that no editor will interact with this data in any way, I would say we could probably skip it for now.

**Comment by swirtSJW** [comment]
There is a bug... or maybe an unrecalled feature where external content it being  inserted at the top of the page, despite the weighting showing that it should come after the title and field office.

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/4c0bb42d-d9b3-47e9-90cd-ae81dc6e2823)

Given the urgency of police AND that no editors will be in this for a long time, I am going to move on and not chase this bug down.

**Comment by davidmpickett** [comment]
Want to note that the bug is only on the Edit screen and not on Node view. If we took edit permissions away from Content Editor, Reviewer, Publisher (which we previously waffled on) on this content type, then that would prevent most editors from seeing this weirdness

**Comment by swirtSJW** [comment]
The bug Dave mentions above is no more.  Resolved.

**Comment by jilladams** [comment]
FAQs already in place. 
Migrations in progress, unblocked now. ETA for code review by ~Fri.

**Comment by swirtSJW** [comment]
Updated upstream data and here is a copy to get into my space 
[police_system_contacts.csv](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13687370/police_system_contacts.csv)

**Comment by swirtSJW** [comment]
The final PR has been merged.  We will just need to run the migration by hand after the code deploys today.

**Comment by swirtSJW** [comment]
The contact migration has been run on prod and all is good
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/e46626bf-755a-4a2c-989c-5a772a1f726b)

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/5fb7a72a-8d86-455d-b702-5d130413814a)

We can call this done.
